composer.json

```diff
@@ -1,8 +1,8 @@
 {
-    "name": "escapestudios/symfony2-coding-standard",
+    "name": "djoos/symfony2-coding-standard",
```

Comment:
Would it be possible to leave this out for the time being? I'll need some more thinking around that as it's [not possible to rename packages on Packagist](https://github.com/composer/packagist/issues/47)...